src/sssom/context.py

```diff
@@ -52,10 +52,13 @@ def _get_built_in_prefix_map() -> Converter:
 HINT = Union[None, PrefixMap, Converter]
 
 
-def ensure_converter(prefix_map: HINT = None) -> Converter:
+def ensure_converter(prefix_map: HINT = None, *, use_bioregistry: bool = True) -> Converter:
```

Comment:
```suggestion
def ensure_converter(prefix_map: HINT = None, *, use_default_prefixmap: bool = True) -> Converter:
```

The "bioregistry" names invites the faulty suggestion that it will actually use bioregistry, rather than a customised context which may or may not be up-to-date in sssom-py.

Reply:
good point